playwright/network.py

```diff
@@ -53,7 +53,7 @@ def postData(self) -> Optional[str]:
         b64_content = self._initializer.get("postData")
         if not b64_content:
             return None
-        return base64.b64decode(bytes(b64_content, "utf-8")).decode()
+        return base64.b64decode(b64_content.encode()).decode()
```

Comment:
Why do we need to encode base64 string into utf8 if b64decode accepts ascii?

Reply:
Probably `base64.b64decode(b64_content).decode()` is enough?

It was more the thought that we have to pass bytes to b64decode.
- b64_content -> str
- b64_content.encode() -> bytes

But seems like a str is also fine: https://docs.python.org/3/library/base64.html#base64.b64decode